charts/vc-authn-oidc/templates/agent/deployment.yaml

```diff
@@ -0,0 +1,131 @@
+{{- $acapyFullName := include "acapy.fullname" . -}}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ $acapyFullName }}
+  labels:
+    {{- include "acapy.labels" . | nindent 4 }}
+spec:
+  {{- if not .Values.acapy.autoscaling.enabled }}
+  replicas: {{ .Values.acapy.replicaCount }}
+  {{- end }}
+  selector:
+    matchLabels:
+      {{- include "acapy.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      {{- with .Values.acapy.podAnnotations }}
+      annotations:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      labels:
+        {{- include "acapy.selectorLabels" . | nindent 8 }}
+    spec:
+      {{- with .Values.acapy.image.pullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      serviceAccountName: {{ include "acapy.serviceAccountName" . }}
+      automountServiceAccountToken: {{ .Values.acapy.serviceAccount.automountServiceAccountToken }}
+      securityContext:
+        {{- toYaml .Values.acapy.podSecurityContext | nindent 8 }}
+      containers:
+        - name: acapy
+          securityContext:
+            {{- toYaml .Values.acapy.containerSecurityContext | nindent 12 }}
+          image: "{{ .Values.acapy.image.repository }}:{{ .Values.acapy.image.tag | default .Chart.AppVersion }}"
+          imagePullPolicy: {{ .Values.acapy.image.pullPolicy }}
+          command:
+          - /bin/bash
+          args: [
+           "-c",
+           "aca-py start \
+           --inbound-transport http '0.0.0.0' {{ .Values.acapy.service.httpPort }} \
+           --outbound-transport http \
+           --admin '0.0.0.0' {{ .Values.acapy.service.adminPort }} \
+           --endpoint {{ include "acapy.agent.url" . }} \
```

Comment:
Could use the `ACAPY_ENDPOINT` environment variable.

Reply:
updated.